src/webgpu/api/validation/encoding/render_bundle.spec.ts

```diff
@@ -236,13 +193,9 @@ g.test('depth_stencil_readonly_mismatch')
     } = t.params;
     await t.selectDeviceForTextureFormatOrSkipTestCase(depthStencilFormat);
 
-    let compatible =
-      bundleDepthReadOnly === passDepthReadOnly && bundleStencilReadOnly === passStencilReadOnly;
-
-    const depthStencilInfo = kTextureFormatInfo[depthStencilFormat];
-    if (depthStencilInfo.depth && depthStencilInfo.stencil) {
-      compatible = compatible && bundleDepthReadOnly === bundleStencilReadOnly;
```

Comment:
Just to check, do we still have test coverage for when bundleDepthReadOnly != bundleStencilReadOnly?

Reply:
Yes: https://github.com/gpuweb/cts/blob/main/src/webgpu/api/validation/encoding/createRenderBundleEncoder.spec.ts#L124